src/components/Menus/SideMenu/FilePane/FilePane.js

```diff
@@ -11,10 +12,10 @@ const FilePane = (props) => {
   const {openFileTab} = props
 
   return (
-    <div className = "file-pane">
+    <ResizableWithHandle className='file-pane' minWidth='200px'>
```

Comment:
❓ another @maxelkins one but a max size would be great so it can't be extended to the whole width